src/Docs/Resources/platform-updates/2019-10-01-breaking-change-validation-service-context.md

```diff
@@ -0,0 +1,3 @@
+[titleEn]: <>(ValidationServiceInterface method signature changes)
+
+We have made changes to the method signature of all ValidationServiceInterface functions by requiring that the last parameter be a SalesChannelContext object and not a Context object. 
```

Comment:
could you also add entries to the CHANGELOG-6.0.md and UPGRADE-6.1.md ?

Reply:
done :)